nxcomp/Log.h

```diff
@@ -138,7 +143,7 @@ class NXLog
 		return ret;
 	}
 
-	std::string stamp_to_string(const NXLogStamp& stamp) const;
+	std::string stamp_to_string(const NXLogStamp& stamp);
```

Comment:
You're reverting this later on... when all of this is done, make sure to squash these "added, removed, added" changes again, please.